frontend/src/App.tsx

```diff
@@ -122,6 +123,11 @@ const App = (): React.ReactElement => {
                     path={`${Routes.VERIFY_EMAIL_PAGE}`}
                     component={VerifyEmail}
                   />
+                  <Route
+                    exact
+                    path={`${Routes.LOGOUT_PAGE}`}
+                    component={Logout}
+                  />
```

Comment:
What's the point of this page? Just for development purposes?

Reply:
Yeah, and also to have a page to direct people to in case something gets screwed up with their auth. So if someone is getting page-breaking auth errors, we can tell them "visit /logout" and try again. (Certain auth errors cause the entire page, including the logout button, to disappear.)